site/load-chart.js

```diff
@@ -5,6 +5,7 @@ google.charts.setOnLoadCallback(setupChart)
 
 const fetchData = fetch('scores.json')
 const embed = location.search === '?embed'
+let dark_mode = window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches
```

Comment:
This can just be `const` as we don't reassign the value.

Reply:
We change it in the event listener if the user changes the preferred color scheme.

Reply:
Ah, I didn't notice that!